torch/csrc/jit/script/compiler.cpp

```diff
@@ -824,6 +873,7 @@ struct to_ir {
     } else if (ident.name() == "print") {
       if (!attributes.empty())
         throw ErrorReport(ident) << "print doesn't accept any keyword arguments";
+      ensureTensors(ident.range(), inputs);
```

Comment:
This is weird because it will underline the `print` part, and not the actual argument.

Reply:
At this point, we've lost the location of the arguments. So we'd have to do refactoring to make it available.